sources/Launcher/DYNRobustnessAnalysisLauncher.cpp

```diff
@@ -380,6 +380,7 @@ RobustnessAnalysisLauncher::simulate(const boost::shared_ptr<DYN::Simulation>& s
       simulation->terminate();
       result.setSuccess(true);
       result.setStatus(CONVERGENCE_STATUS);
+      simulation->dumpIIDMFile(result.getOutputIIDMStream());
```

Comment:
Why is this done here and not in the same place than the others? Also the iidm should not be written if finalState_.iidmFile_  is false on dynawo side (in this case this stream should remain empty)